hugegraph-dist/src/assembly/static/bin/hugegraph-server.sh

```diff
@@ -66,6 +66,11 @@ MAX_MEM=$((32 * 1024))
 MIN_MEM=$((1 * 512))
 MIN_JAVA_VERSION=8
 
+# download binary file
+if [[ ! -e ${CONF}/hugegraph-server.keystore  ]]; then
+  wget -P "${CONF}" https://github.com/apache/incubator-hugegraph-doc/blob/master/dist/server/hugegraph-server.keystore
```

Comment:
TODO: we should better use a tag branch like `1.0.0` to avoid `master` branch changed uri